package.json

```diff
@@ -0,0 +1,101 @@
+  "main": "index.js",
+  "scripts": {
+    "start": "babel-node server/index.js",
+    "test": "gulp run-tests",
+    "build": "webpack --config ./webpack.prod.config.js --progress --colors"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/victor4l/PostIt.git"
+  },
+  "keywords": [
+    "PostIt",
+    "Notifications",
+    "Post",
+    "It"
+  ],
+  "author": "Victor Idongesit",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/victor4l/PostIt/issues"
+  },
+  "homepage": "https://github.com/victor4l/PostIt#readme",
+  "devDependencies": {
+    "browser-sync": "^2.18.12",
+    "eslint": "^3.19.0",
+    "eslint-config-airbnb": "^15.0.1",
+    "eslint-config-airbnb-base": "^11.2.0",
+    "eslint-plugin-import": "^2.3.0",
+    "eslint-plugin-jsx-a11y": "^5.0.3",
+    "eslint-plugin-react": "^7.2.0",
+    "gulp-sass": "^3.1.0",
+    "webpack-dev-server": "^2.5.1"
+  },
+  "dependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.25.0",
+    "babel-loader": "^7.0.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "babel-register": "^6.24.1",
+    "bcrypt-nodejs": "0.0.3",
+    "body-parser": "^1.17.2",
+    "cors": "^2.8.4",
+    "css-loader": "^0.28.4",
+    "dotenv": "^4.0.0",
+    "eslint-config-airbnb": "^15.0.1",
+    "eslint-config-airbnb-base": "^11.2.0",
+    "express": "^4.15.3",
+    "extract-text-webpack-plugin": "^2.1.2",
+    "file-loader": "^0.11.2",
+    "font-awesome": "^4.7.0",
+    "gulp": "^3.9.1",
+    "gulp-babel": "^6.1.2",
+    "gulp-babel-istanbul": "^1.6.0",
+    "gulp-coveralls": "^0.1.4",
+    "gulp-inject-modules": "^1.0.0",
+    "gulp-jasmine-node": "^1.0.7",
+    "jasmine": "^2.6.0",
+    "jquery": "^3.2.1",
+    "jsonwebtoken": "^7.4.2",
+    "jwt-decode": "^2.2.0",
+    "material-design-icons": "^3.0.1",
+    "material-icons": "^0.1.0",
+    "materialize-css": "^0.98.2",
+    "morgan": "^1.8.2",
+    "nexmo": "^2.0.2",
+    "nodemailer": "^4.0.1",
+    "passport": "^0.3.2",
+    "passport-google-oauth2": "^0.1.6",
+    "passport-jwt": "^2.2.1",
+    "passport-local": "^1.0.0",
+    "pg": "^6.2.4",
+    "pg-hstore": "^2.3.2",
+    "raw-loader": "^0.5.1",
+    "react": "^15.6.1",
+    "react-dom": "^15.6.1",
+    "react-google-login": "^2.9.3",
+    "react-history": "^0.18.2",
+    "react-notification-system": "^0.2.14",
+    "react-paginate": "^4.4.3",
+    "react-redux": "^5.0.5",
+    "react-router": "^4.1.1",
+    "react-router-dom": "^4.1.1",
+    "redux": "^3.7.2",
+    "sass-loader": "^6.0.5",
+    "sequelize": "^4.1.0",
+    "socket.io": "^2.0.3",
+    "socket.io-client": "^2.0.3",
+    "style-loader": "^0.18.2",
+    "superagent": "^3.5.2",
+    "supertest": "^3.0.0",
+    "url-loader": "^0.5.9",
+    "webpack": "^2.6.1",
+    "webpack-dev-middleware": "^1.12.0",
+    "webpack-hot-middleware": "^2.18.2"
+  }
+}
```

Comment:
What is the purpose of using gulp and webpack together? why not use only webpack.